test/unit/math/prim/mat/prob/categorical_logit_test.cpp

```diff
@@ -79,3 +81,58 @@ TEST(ProbDistributionsCategoricalLogit, error) {
   ns[1] = 12;
   EXPECT_THROW(categorical_logit_log(ns, theta), std::domain_error);
 }
+
+TEST(ProbDistributionsCategoricalLogit, error_check) {
```

Comment:
In general (though this is not always so in the code, as you found) we try to do one test file per one implementation file so could you break your own tests out into `categorical_logit_rng_test.cpp`? It makes it clearer what is and isn't tested.